rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/dao/repository/RecruitmentPeriodRepository.java

```diff
@@ -3,5 +3,16 @@
 import org.niewidoczniakademicy.rezerwacje.core.model.database.RecruitmentPeriod;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.time.LocalDate;
+import java.util.List;
+
 public interface RecruitmentPeriodRepository extends JpaRepository<RecruitmentPeriod, Long> {
-}
+
+    @Override
+    <S extends RecruitmentPeriod> S save(S entity);
+
+    @Override
+    List<RecruitmentPeriod> findAll();
+
+    List<RecruitmentPeriod> findByStartDateGreaterThanEqualAndEndDateLessThanEqual(LocalDate startDate, LocalDate endDate);
```

Comment:
Może nazwać tą metodę ```findByPeriodOfTime``` albo coś takiego?

Reply:
Właśnie chodzi o to że Spring generuje odpowiednie zapytanie SQL na podstawie nazwy. Ewentualnie można zmienić nazwę tak jak mówisz, ale wtedy zapytanie trzeba by było napisać ręcznie w adnotacji

Reply:
Ok